pkgs/development/python-modules/httpagentparser/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "httpagentparser";
+  version = "1.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "73Y9MZk912GCWs7myLNL4yuVzxZ10cc8PNNfnlKDGyY=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  # PyPi version does not include test directory
+  doCheck = false;
+
+  pythonImportsCheck = [ "httpagentparser" ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/httpagentparser/";
```

```suggestion
    homepage = "https://github.com/shon/httpagentparser";
```

The URL on PyPI is not longer valid.